tests/test_untrained_model.py

```diff
@@ -0,0 +1,29 @@
+import tensorflow as tf
+import numpy as np
+from src.model import create_model
+from constants import IMG_LEN, NUM_CHANNELS
+
+
+def test_model_compilation():
+    model = create_model()
+    model.compile(
+        optimizer="adam", loss="binary_crossentropy", metrics=["accuracy"]
+    )
+    assert model.optimizer is not None, "Model optimizer should be set."
+    assert model.loss is not None, "Model loss should be set."
```

Comment:
Ten test trochę niczego nie testuje. Sorki.
Gdybyśmy mieli OOP i własną funkcję compile_model() to wtedy byłoby co testować.
Teraz to ten test testuje nie nasz kod tylko metodę compile z tensorflow.

Reply:
będzie OOP